Fw/Comp/QueuedComponentBase.cpp

```diff
@@ -33,7 +33,7 @@ namespace Fw {
         queueName = this->m_objName;
 #else
         char queueNameChar[FW_QUEUE_NAME_MAX_SIZE];
-        (void)snprintf(queueNameChar,sizeof(queueNameChar),"CompQ_%d",Os::Queue::getNumQueues());
+        (void)snprintf(queueNameChar,sizeof(queueNameChar),"CompQ_%d",this->m_queue.getNumQueues());
```

Comment:
Does this still compile with 1.5.0?   getNumQueues is now a static class function is is therefore traditionally called with:
```
Os::Queue::getNumQueues()
```

Reply:
Ah ok, I had not actually tested against `devel` so I didn't realize @Joshua-Anderson had turned `getNumQueues` into a static function. That seems like the better fix anyway!